aiidalab_widgets_base/codes.py

```diff
@@ -20,27 +21,32 @@ def valid_aiidacode_args(arguments):
 
 class CodeDropdown(ipw.VBox):
     """Code selection widget."""
```

Comment:
Please document the use of the `selected_code` traitlet here

Reply:
done.